pipelines/skylab/snm3C/snm3C.wdl

```diff
@@ -248,7 +277,6 @@ task Demultiplexing {
     for i in $(seq 1 "${batch_number}"); do
         tar -cf - $WORKING_DIR/batch${i}/*.fq.gz | pigz > ~{plate_id}.${i}.cutadapt_output_files.tar.gz
     done
-  >>>
```

Comment:
@aawdeh , i think this accidentally got deleted. maybe try adding this back and things will start working?

Reply:
Ah, thank you!